app/components/post_draft/draft_input/index.tsx

```diff
@@ -228,10 +258,20 @@ export default function DraftInput({
                             testID={sendActionTestID}
                             disabled={sendActionDisabled}
                             sendMessage={handleSendMessage}
+                            showScheduledPostOptions={handleShowScheduledPostOptions}
                         />
                     </View>
                 </ScrollView>
             </SafeAreaView>
         </>
     );
 }
+
+const enhanced = withObservables([], ({database}: WithDatabaseArgs) => {
```

Comment:
Should we move this to `app/components/post_draft/draft_input/index.ts` and rename this file `app/components/post_draft/draft_input/draft_input.tsx` as we follow this pattern throughout the repo?

Reply:
Done.,